lib/MusicBrainz/Server/Data/Role/EntityModelClass.pm

```diff
@@ -0,0 +1,42 @@
+package MusicBrainz::Server::Data::Role::EntityModelClass;
```

Comment:
Requiring every entity to be in entities.json would seem good to me. I recently fixed a warning because of track there missing mbid.relatable entirely, so I'd actually even prefer us to have a fairly strict model of how entities need to look like and what they should have (set to false if they don't actually have it). But making sure they're all there would be a good start.

Reply:
@mwiencek: I don’t see the added value of requiring this definition to be in JSON if it isn’t consumed outside of `MusicBrainz::Server::Data`.

@reosarevok: It seems absurd to do that: There are 302 entity types in this JSON file and 32 different possible subkeys; that would make 9664 keys, paradoxically for no value either.

Having a documentation for this JSON file would be more helpful.